### PR TITLE
make it a proper python package

### PR DESCRIPTION
**Pull request by peerchemist**
This will make it pypi ready and usable as a lib.

### CODE REVIEW

**Comment by peerchemist** [comment]
I prefer to use absolute imports.

**Comment by SheldonYS** [comment]
The `Install` part in `README.md` needs to be updated as well.

File: pyvsystems/account.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-from .setting import *
-from .crypto import *
-from .error import *
-from .words import WORDS
+from pyvsystems.setting import *
+from pyvsystems.crypto import *
+from pyvsystems.error import *
+from pyvsystems.words import WORDS
 from . import is_offline
 from . import default_chain
```
Comment (Icermli) [inline pyvsystems/account.py]: If you prefer absolute import, please make all absolute. I find plenty of relative import

**Comment by peerchemist** [comment]
Will redo README.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    version="0.1.0",
+    description="A python wrapper for vsys api.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    keywords=["api wrapper", "blockchain", "vsystems", "smart contract", "supernode", "defi"],
+    url="https://github.com/virtualeconomy/pyvsystems",
+    author="V SYSTEMS",
+    author_email="info@v.systems",
+    license="MIT",
+    packages=["pyvsystems"]
```
Comment (SheldonYS) [inline setup.py]: Need add `install_requires` in setup. As the following line:
```
install_requires = ["requests", "python-axolotl-curve25519", "pyblake2", "base58"],
```

Comment (SheldonYS) [reply setup.py]: tested failed with `python 2.7`. Need add `python_requires='>=3'` in setup.

**Comment by SheldonYS** [comment]
I suggest to use `relative import` for this library. Because `relative import` can be used with both `PyPi` and `git` (`git clone` or `git submodule`) command with specific folder. But `absolute import` only can be used with `PyPi`. If someone want to fork with git and change some code, he needs to change a lots of lines.

**Comment by peerchemist** [comment]
It can also be used like this:

in requirements.txt

> git+https://github.com/peerchemist/pyvsystems.git

But ok, we can do relative.

**Comment by peerchemist** [comment]
Is it worth to upkeep the py2.7 compatibility? It's EOL.

**Comment by SheldonYS** [comment]
> Is it worth to upkeep the py2.7 compatibility? It's EOL.

No need to keep the py2.7 compatibility. We need add `python_requires='>=3'` in `setup.py` to enforce user using this lib under python 3 or above environment. If user use py2.7, the pip setup will be failed.

**Pull request merged**